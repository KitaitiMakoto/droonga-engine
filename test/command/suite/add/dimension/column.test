#@require-catalog-version 2
{
  "type": "add",
  "dataset": "Default",
  "body": {
    "table": "Products",
    "key": "Groonga",
    "values": {
      "category": "groonga"
    }
  }
}
{
  "type": "add",
  "dataset": "Default",
  "body": {
    "table": "Products",
    "key": "Rroonga",
    "values": {
      "category": "groonga"
    }
  }
}
{
  "type": "add",
  "dataset": "Default",
  "body": {
    "table": "Products",
    "key": "Ruby",
    "values": {
      "category": "ruby"
    }
  }
}
{
  "type": "search",
  "dataset": "Default",
  "body": {
    "queries": {
      "products": {
        "source": "Products",
        "sortBy": ["_key"],
        "output": {
          "elements": ["records"],
          "attributes": ["_key", "category"],
          "limit": -1
        }
      }
    }
  }
}
