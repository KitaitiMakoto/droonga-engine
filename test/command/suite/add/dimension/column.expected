[
  "droonga.message",
  0,
  {
    "inReplyTo": "request-id",
    "statusCode": 200,
    "type": "add.result",
    "body": true
  }
]
[
  "droonga.message",
  0,
  {
    "inReplyTo": "request-id",
    "statusCode": 200,
    "type": "add.result",
    "body": true
  }
]
[
  "droonga.message",
  0,
  {
    "inReplyTo": "request-id",
    "statusCode": 200,
    "type": "add.result",
    "body": true
  }
]
[
  "droonga.message",
  0,
  {
    "inReplyTo": "request-id",
    "statusCode": 200,
    "type": "search.result",
    "body": {
      "products": {
        "records": [
          [
            "Groonga",
            "groonga"
          ],
          [
            "Rroonga",
            "groonga"
          ],
          [
            "Ruby",
            "ruby"
          ]
        ]
      }
    }
  }
]
